RFCs/FS-1128-better-interpolated-triple-quoted-strings.md

```diff
@@ -21,99 +22,117 @@ let json = $$"""
+*It should work, changes will hopefully be limited to lexing stage*
+- If this is a change or extension to FSharp.Core, what happens when previous versions of the F# compiler encounter this construct?
 *It is not a change/extension to FSharp.Core*
 
 # Pragmatics
 
 ## Diagnostics
 
-* New compiler error for interpolated string literals with too many consecutive `{` or `}` characters.
-* Otherwise, same compiler errors as for regular (single-`$`) interpolated strings.
+- New compiler error for interpolated string literals with too many consecutive `{` or `}` characters.
+- Otherwise, same compiler errors as for regular (single-`$`) interpolated strings.
 
 ## Tooling
 
-Compiler tooling should offer the same support for the extended form of interpolated string literals as it currently does for interpolated triple quoted literals. This includes colorization, autocompletion, and navigational features in an interpolated context.
+Compiler tooling should offer the same support for the extended form of interpolated string literals as it currently does for interpolated triple quoted literals.
+This includes colorization, autocompletion, and navigational features in an interpolated context.
 
 ## Performance
 
-AFAICT there should be no impact on performance (although there might be if using certain constructs in regexes within .fsl files can have a big impact `XXX TODO`)
+This feature should have no impact on performance.
 
 ## Scaling
 
-Please list the dimensions that describe the inputs for this new feature, e.g. "number of widgets" etc.  For each, estimate a reasonable upper bound for the expected size in human-written code and machine-generated code that the compiler will accept.
-
-For example
-
-* Expected maximum number of widgets in reasonable hand-written code: 100
-* Expected reasonable upper bound for number of widgets accepted: 500
-
-Testing should particularly check that compilation is linear (or log-linear or similar) along these dimensions.  If quadratic or worse this should ideally be noted in the RFC.
-
-`TODO` *No clue really, but I expect more than 3x$ will be very rarely used though*
+We do not set an explicit limit on the number of `$` characters allowed at the beginning of a string literal.
+In a hand-written code it will typically not exceed 3.
 
 ## Culture-aware formatting/parsing
 
-Does the proposed RFC interact with culture-aware formatting and parsing of numbers, dates and currencies? For example, if the RFC includes plaintext outputs, are these outputs specified to be culture-invariant or current-culture.
-
-*No* `TODO: remove this section?`
+N/A
 
 # Unresolved questions
 
-1. Do we want to also provide a way to easily input procent signs in interpolated string literals?
+1. Do we want to also provide a way to easily input percent signs in interpolated string literals?
+2. Should we extend this feature to regular (single-quoted) string literals as well in addition to triple quoted strings?
```

Comment:
For convenience, and also by the time we reach a consensus, can you add a link to the specific comment in the discussion thread that asks this question or (quite definitively) answers it?

Reply:
Done